src/fsharp/service/ServiceUntypedParse.fs

```diff
@@ -600,7 +600,7 @@ type FSharpParseFileResults(errors: FSharpDiagnostic[], input: ParsedInput optio
                           yield! walkExpr true e ]
             
             // Process a class declaration or F# type declaration
-            let rec walkTycon (TypeDefn(ComponentInfo(_, _, _, _, _, _, _, _), repr, membDefns, m)) =
+            let rec walkTycon (TypeDefn(ComponentInfo(_, _, _, _, _, _, _, _), repr, membDefns, _, m)) =
```

Comment:
This should walk the implicit ctor if it's there

Reply:
I expect it to walk in it for object model types if it already does, this behaviour hasn't changed. Do you mean, it should walk in it for types that don't allow implicit constructors too?  This PR only adds thrown constructors back to non-object model types.